charts/retool/values.yaml

```diff
@@ -329,12 +329,15 @@ workflows:
     # Scaling this number will increase the number of workflow workers, e.g. a replicaCount of 4
     # will launch 9 pods -- 1 workflow backend, 4 workflow workers, and 4 for temporal cluster
     replicaCount: 1
+    # portName: custom-port-name, e.g. http-workflows-worker; defaults to "retool.name"
+    # metricsPortName: custom-port-name, e.g. http-workflow-worker-metrics; defaults to "metrics"
```

Comment:
For all the defaults in our values.yaml, I think it would make sense to just pick a default port name and possibly just drop the `template "retool.name" .` and `| default "metrics"` dynamic fallbacks. The defaults can be static (i.e. no point to using `template "retool.name"` anyway) and we can even just use the istio naming convention for them. This would remove the need for those conditionals in templates and just simplify a bit. Adding port names is a non-breaking change anyway I am pretty sure, so extremely low risk.